.github/workflows/python-checks.yml

```diff
@@ -43,23 +43,18 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
-          # Manually install CPU-only version of torch so we're not carrying around giant GPU drivers/kernels
-          python -m pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/cpu
-          python -m pip install -e "s3torchconnectorclient[test]"
-          python -m pip install -e "s3torchconnector[test]"
-          python -m pip install -e "s3torchbenchmarking[test]"
+          python -m pip install "./s3torchconnectorclient[test]"
+          python -m pip install "./s3torchconnector[test]"
 
-      - name: s3torchconnectorclient unit tests
+      - name: Run s3torchconnectorclient unit tests
         run: pytest s3torchconnectorclient/python/tst/unit --hypothesis-profile ci --hypothesis-show-statistics -c ./
-      - name: s3torchconnector unit tests
+      - name: Run s3torchconnector unit tests
         run: pytest s3torchconnector/tst/unit --ignore s3torchconnector/tst/unit/lightning --hypothesis-profile ci --hypothesis-show-statistics -c ./
-      - name: s3torchbenchmarking unit tests
```

Comment:
Do we have any conflicts because of s3torchbenchmarking unit tests? I know that they essentially do nothing at the moment, but if we will delete them now, we can forget to add them back later.